content/pt/docs/reference/access-authn-authz/authentication.md

```diff
@@ -0,0 +1,889 @@
+Isto é projetado para o uso em combinação com um proxy de autenticação, o qual irá atribuir o valor do cabeçalho da requisição.
+ 
+* `--requestheader-username-headers` Obrigatório, nao faz distinção entre caracteres maiúsculos/minúsculos. Nomes de cabeçalhos a serem verificados, em ordem, para a identidade do usuário. O primeiro cabeçalho contendo um valor será usado para o nome do usuário.
+* `--requestheader-group-headers` 1.6+. Opcional, não faz distinção entre caracteres maiúsculos/minúsculos. "X-Remote-Group" é recomendado. Nomes de cabeçalhos a serem verificados, em ordem, para os grupos do usuário. Todos os valores especificados em todos os cabeçalhos serão utilizados como nome dos grupos do usuário.
+ 
+* `--requestheader-extra-headers-prefix` 1.6+. Opcional, não faz distinção entre caracteres maiúsculos/minúsculos. "X-Remote-Extra-" é recomendado. Prefixos de cabeçalhos a serem buscados para determinar informações extras sobre o usuário (normalmente utilizado pelo plugin de autorização configurado). Todos os cabeçalhos que começam com qualquer um dos prefixos especificados têm o prefixo removido. O restante do nome do cabeçalho é transformado em letra minuscula, decodificado [percent-decoded](https://tools.ietf.org/html/rfc3986#section-2.1) e torna-se uma chave extra, e o valor do cabeçalho torna-se um valor extra.
+ 
+{{< note >}}
+Antes da versão 1.11.3 (e 1.10.7, 1.9.11), a chave extra só poderia conter caracteres os quais fossem [legais em rótulos de cabeçalhos HTTP](https://tools.ietf.org/html/rfc7230#section-3.2.6).
+{{< /note >}}
+ 
+Por exemplo, com esta configuração:
+ 
+```
+--requestheader-username-headers=X-Remote-User
+--requestheader-group-headers=X-Remote-Group
+--requestheader-extra-headers-prefix=X-Remote-Extra-
+```
+ 
+e esta requisição:
+ 
+```http
+GET / HTTP/1.1
+X-Remote-User: fido
+X-Remote-Group: dogs
+X-Remote-Group: dachshunds
+X-Remote-Extra-Acme.com%2Fproject: some-project
+X-Remote-Extra-Scopes: openid
+X-Remote-Extra-Scopes: profile
+```
+ 
+resultaria nesta informação de usuário:
+ 
+```yaml
+name: fido
+groups:
+- dogs
+- dachshunds
+extra:
+ acme.com/project:
+ - some-project
+ scopes:
+ - openid
+ - profile
+```
+ 
+Para prevenir falsificação de cabeçalhos, o proxy de autenticação deverá apresentar um certificado de cliente válido para o servidor de API para que possa ser validado diante da autoridade de certificados (CA) antes que os cabeçalhos de requisições sejam verificados. AVISO: **não** re-utilize uma autoridade de certificados (CA) que seja utilizado em um contexto diferente ao menos que você entenda os riscos e os mecanismos de proteção da utilização de uma autoridade de certificados.
+ 
+* `--requestheader-client-ca-file` Obrigatório. Pacote de certificados no formato PEM . Um certificado válido deve ser apresentado e validado diante da autoridade de certificados no arquivo especificado antes da verificação de cabeçalhos de requisição para os nomes do usuário.
```

Comment:
`* `--requestheader-client-ca-file` Obrigatório. Pacote de certificados no formato PEM .`

remove the extra line between `no formato PEM` and `.`

Reply:
I would write

...Um certificado válido deve ser apresentado e validado com a autoridade de certificados no arquivo especificado